staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2629,6 +2630,12 @@ type PodAffinityTerm struct {
 	// selected pods is running.
 	// Empty topologyKey is not allowed.
 	TopologyKey string `json:"topologyKey" protobuf:"bytes,3,opt,name=topologyKey"`
+	// Selects Namespaces using cluster scoped-labels.  This
+	// matches namespaces selected by this label selector.
+	// This field follows standard label selector semantics.
+	// If present but empty, this selector selects all namespaces.
```

Comment:
Please update this comment as well and update the generated files. The rest LGTM.